src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -25,7 +27,8 @@
     /**
      * Used for initial separation of command word and args.
      */
-    private static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
+    private static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<type>[A-Z]\\s+)?"
```

Comment:
I think you pattern did not match our command. Here is my suggestion: 
`(?<type>[A-Z]) (?<commandWord>\S+) (?<arguments>.*)`

Reply:
Ok I tried to change to this but err java gives me an error: Illegal escape character in String literal

Reply:
Ahh I see, just add another '\' before each of the current '\'